setup.py

```diff
@@ -9,7 +9,7 @@
 with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
     long_description = f.read()
 
-requires = ['Sphinx>=1.6.3']
+requires = ['Sphinx>=1.6.3', 'cairosvg>=1.0']
```

Comment:
Please declare CairoSVG as an optional dependency as explained in the [Setuptools documentation](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies).